cmd/root_test.go

```diff
@@ -473,6 +473,74 @@ func TestCache(t *testing.T) {
 		stats.Formatted: 32,
 		stats.Changed:   0,
 	})
+
+	// test how cache behaves when there are formatting errors
+
+	cfg = &config.Config{
+		FormatterConfigs: map[string]*config.Formatter{
+			"echo": {
+				Command:  "echo",
+				Includes: []string{"*.hs"},
+			},
+			"fail": {
+				Command:  "touch",
+				Options:  []string{"--bad-arg"},
+				Includes: []string{"*.py"},
+			},
+		},
+	}
+	test.WriteConfig(t, configPath, cfg)
+
+	// formatters have changed, so the cache is busted and we start fresh
```

Comment:
I'm confused by this comment. If we're starting fresh, why aren't we erroring out trying to format `*.py` files?

Reply:
I'll clear up the comments in that test and make it clearer what it's expecting to happen